assets/src/design-system/components/contextMenu/stories/index.js

```diff
@@ -167,3 +190,106 @@ export const Animated = () => {
     </ViewportContainer>
   );
 };
+
+const blankPageItems = [
```

Comment:
I went ahead and hardcoded all of these since it's just dummy code to make sure we have our UI bases covered ahead of functionality. This will all go into the editor and be translated there.